VEDA_OS01/models.py

```diff
@@ -11,6 +11,14 @@ def _createHex():
     return uuid.uuid1().hex
 
 
+class VideoTranscriptStatus(object):
```

Comment:
@muhammad-ammar I've added these statuses into `VideoStatus` in models, it only needs rebasing to use from there.